conans/model/layout.py

```diff
@@ -0,0 +1,90 @@
+import os
+
+
+class _LayoutEntry(object):
+
+    def __init__(self):
+        self.folder = ""
+        self.includedirs = {}
```

Comment:
The model dict {"folder": "file patterns list"} seems a bit weird to me. I think it would be more explicit to have:

- Lists of folders
- Patterns ``lib_patterns``, ``bin_patterns`` with lists of files

Reply:
Check now